connmgr/seed.go

```diff
@@ -67,3 +74,75 @@ func SeedFromDNS(dnsSeeds []string, defaultPort uint16, reqServices wire.Service
 		}(host)
 	}
 }
+
+type Node struct {
```

Comment:
I don't think this needs to be exported.  It's not needed or used outside of the package.